pkg/unity/process-line-mappings.go

```diff
@@ -0,0 +1,45 @@
+package unity
+
+import (
+	"github.com/bugsnag/bugsnag-cli/pkg/log"
+	"github.com/bugsnag/bugsnag-cli/pkg/options"
+	"github.com/bugsnag/bugsnag-cli/pkg/server"
+	"github.com/bugsnag/bugsnag-cli/pkg/utils"
+)
+
+func UploadAndroidLineMappings(
+	lineMappingFile string,
+	soBuildId string,
+	endpoint string,
+	options options.CLI,
+	manifestData map[string]string,
+	logger log.Logger,
+) error {
+	opts := utils.UnityLineMappingOptions{
+		APIKey:         manifestData["apiKey"],
+		AppID:          manifestData["applicationId"],
+		AppVersion:     manifestData["versionName"],
+		AppVersionCode: manifestData["versionCode"],
+		SOBuildID:      soBuildId,
+		ProjectRoot:    options.Upload.UnityAndroid.ProjectRoot,
+		Overwrite:      options.Upload.UnityAndroid.Overwrite,
+	}
```

Comment:
This pattern of having a new struct seems a little convoluted as you have to do the conversion here, just for the build function to take the values out and put them into a map immediately afterwards.

I suggest keeping the existing pattern, at the expense of a relatively long parameter list.